syntax = "proto3";

package server_services;

message ConvertCsvToDbAndSaveRequest {
    string file_name = 1; // Nome do arquivo CSV
}

message ConvertCsvToDbAndSaveResponse {
    bool success = 1;
    string message = 2;
}

service FileConversionService {
    rpc ConvertCsvToDbAndSave (ConvertCsvToDbAndSaveRequest) returns (ConvertCsvToDbAndSaveResponse);
}
